contracts/adapters/bzx/BzxStakingAdapter.sol

```diff
@@ -0,0 +1,104 @@
+// Copyright (C) 2020 Zerion Inc. <https://zerion.io>
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see <https://www.gnu.org/licenses/>.
+
+pragma solidity 0.6.5;
+pragma experimental ABIEncoderV2;
+
+import {ERC20} from "../../ERC20.sol";
+import {ProtocolAdapter} from "../ProtocolAdapter.sol";
+
+/**
+ * @dev StakingRewards contract interface.
+ * Only the functions required for AragonStakingAdapter contract are added.
+ * The StakingRewards contract is available here
+ * github.com/Synthetixio/synthetix/blob/master/contracts/StakingRewards.sol.
+ */
+interface StakingRewards {
+
+    function balanceOfByAssets(address account)
+        external
+        view
+        returns (
+            uint256 bzrxBalance,
+            uint256 iBZRXBalance,
+            uint256 vBZRXBalance,
+            uint256 lPTokenBalance
+        );
+
+    function balanceOfByAsset(address token, address account)
+        external
+        view
+        returns (uint256 balance);
+}
+
+/**
+ * @title Adapter for BZX protocol (staking).
+ * @dev Implementation of ProtocolAdapter interface.
+ * @author Roman Iftodi <romeo8881@gmail.com>
+ */
+contract BzxStakingAdapter is ProtocolAdapter {
```

Comment:
Change name

Reply:
I changed the other one to BzxVestingStakingRewards and this left unchanged